lgtm.py

```diff
@@ -35,6 +35,7 @@ def _make_lgtm_get(self, url: str) -> dict:
         )
         return r.json()
 
+    # Retrieves a user's projects
```

Comment:
Can you use the python standard for API doc comments?

Reply:
Can you point me in that direction. I can google it but want to make sure we are on the same page. I'm not native to python :/

Reply:
@JLLeitschuh 

Can you send that link to me? I tried looking online but can't find any defacto API doc comment rules.

Reply:
I think this covers it.

https://www.programiz.com/python-programming/docstrings